metricbeat/module/system/filesystem/helper.go

```diff
@@ -112,8 +115,10 @@ func filterFileSystemList(fsList []sigar.FileSystem) []sigar.FileSystem {
 // GetFileSystemStat retreves stats for a single filesystem
 func GetFileSystemStat(fs sigar.FileSystem) (*FSStat, error) {
 	stat := sigar.FileSystemUsage{}
-	if err := stat.Get(fs.DirName); err != nil {
-		return nil, err
+	if fs.SysTypeName != UnavailableDisk {
+		if err := stat.Get(fs.DirName); err != nil {
+			return nil, err
+		}
 	}
```

Comment:
I wonder if we should just ignore this error if we want to always report some event even if these stats are not available.
```suggestion
	if fs.SysTypeName != UnavailableDisk {
		if err := stat.Get(fs.DirName); err != nil {
			log.Debugf("....: %v", err)
		}
	}
```